tests/ui/process/process-sigpipe.rs

```diff
@@ -26,7 +26,11 @@ fn main() {
         thread::sleep_ms(5000);
         process::exit(1);
     });
-    let output = process::Command::new("sh").arg("-c").arg("yes | head").output().unwrap();
+    #[cfg(not(target_os = "nto"))]
+    let command = "yes | head";
+    #[cfg(target_os = "nto")]
+    let command = "awk 'BEGIN {while (1) print yes}' | awk 'BEGIN {exit}'";
```

Comment:
If `head` is there, `yes` could be emulated with `sh -c 'while :; do echo y; done'`.

Reply:
[Head is available](https://www.qnx.com/developers/docs/7.1/#com.qnx.doc.neutrino.utilities/topic/h/head.html)! 
Should we call the command `sh -c 'while :; do echo y; done' | head` on all systems (unconditionally)?

Reply:
Not for me to decide, but can't think of any downsides. With `yes` removed, the test would only rely on POSIX commands.

Reply:
Sounds fine to me.

Reply:
@mvf I slightly changed the command so that it actually stops looping: `while echo y ; do : ; done | head`. 
I decided against removing the original version (using `yes`) in this pull request -- too often I've observed that some tests fail in integration which are not run in pull requests. I will remove it separately.

Reply:
> [...] so that it actually stops looping

This means that on the QNX shell, `echo` is not a built-in and `/bin/echo` gets the `SIGPIPE` (instead of the shell itself). This causes it to return non-zero, which was ignored in the loop body. Making `echo` the condition is fine.

Reply:
Per the [QNX Neutrino docs on echo](https://www.qnx.com/developers/docs/7.1/index.html#com.qnx.doc.neutrino.utilities/topic/e/echo.html):
> a shell builtin command (see the [echo](https://www.qnx.com/developers/docs/7.1/com.qnx.doc.neutrino.utilities/topic/k/ksh.html#ksh__echo) command for [ksh](https://www.qnx.com/developers/docs/7.1/com.qnx.doc.neutrino.utilities/topic/k/ksh.html))
>    the echo utility, which is a standalone executable that can operate without the availability of the system shell

So echo apparently is in fact a shell builtin? But I don't know if `sh` is a symlink to `ksh` on QNX's default installations.